cgminer.c

```diff
@@ -1588,6 +1597,48 @@ static struct opt_table opt_config_table[] = {
 		     set_int_0_to_100, opt_show_intval, &opt_bxm_bits,
 		     "Set BXM bits for overclocking"),
 #endif
+#ifdef USE_BITFURY16
+	OPT_WITHOUT_ARG("--bf16-set-clock",
+			opt_set_bool, &opt_bf16_set_clock,
```

Comment:
Build with other drivers fails here onwards with:
cgminer.c:1602:19: error: ‘opt_bf16_set_clock’ undeclared here (not in a function)
    opt_set_bool, &opt_bf16_set_clock,

with the following generic configure:
./configure --enable-avalon --enable-avalon2 --enable-avalon4 --enable-avalon7 --enable-bflsc --enable-bitforce --enable-bitfury --enable-bitfury16 --enable-blockerupter --enable-cointerra --enable-drillbit --enable-hashfast --enable-hashratio --enable-icarus --enable-klondike --enable-modminer